OrbitGl/TextRenderer.cpp

```diff
@@ -270,6 +270,85 @@ void TextRenderer::AddText( const char* a_Text
     AddTextInternal( m_Font, a_Text, ColorToVec4( a_Color ), &m_Pen, a_MaxSize, a_Z );
 }
 
+//-----------------------------------------------------------------------------
+void TextRenderer::AddTextTrailingTimePrioritized( const char* a_Text
```

Comment:
Function name could be generalized to something not related to time.  AddTextTrailingCharsPrioritized maybe?

Reply:
AddTextTrailingCharsPrioritized sounds good to me!

Reply:
cool!